polynote-kernel/src/main/scala/polynote/kernel/interpreter/python/PythonInterpreter.scala

```diff
@@ -1,29 +1,39 @@
 package polynote.kernel.interpreter
 package python
-import java.nio.file.{Files, Path, Paths}
+import java.io.{File, FileReader, PrintWriter, StringWriter}
+import java.nio.charset.StandardCharsets
+import java.nio.file.{Files, Path, Paths, StandardOpenOption}
 import java.util
-import java.util.concurrent.atomic.{AtomicBoolean, AtomicReference}
-import java.util.concurrent.{Executors, LinkedBlockingQueue, ThreadFactory}
+import java.util.concurrent.atomic.AtomicReference
+import java.util.concurrent.{Executors, ThreadFactory}
 
+import cats.effect.concurrent.Ref
 import cats.syntax.traverse._
+import cats.syntax.either._
 import cats.instances.list._
+import io.circe.generic.semiauto._
+import io.circe.syntax._
+import io.circe._
+import io.circe.yaml.syntax._
 import jep.python.{PyCallable, PyObject}
-import jep.{Jep, JepConfig, JepException, MainInterpreter, NamingConventionClassEnquirer, SharedInterpreter, SubInterpreter}
+import jep.{Jep, JepConfig, JepException, NamingConventionClassEnquirer, SharedInterpreter}
 import polynote.config
 import polynote.config.{PolynoteConfig, pip}
 import polynote.kernel.environment.{Config, CurrentNotebook, CurrentRuntime, CurrentTask}
+import polynote.kernel.logging.Logging
 import polynote.kernel.task.TaskManager
-import polynote.kernel.{CompileErrors, Completion, CompletionType, InterpreterEnv, KernelReport, ParameterHint, ParameterHints, Pos, ResultValue, ScalaCompiler, Signatures}
+import polynote.kernel.{BaseEnv, CompileErrors, Completion, CompletionType, GlobalEnv, InterpreterEnv, KernelReport, ParameterHint, ParameterHints, ResultValue, ScalaCompiler, Signatures, TaskInfo}
 import polynote.messages.{CellID, Notebook, NotebookConfig, ShortString, TinyList, TinyString}
 import polynote.runtime.python.{PythonFunction, PythonObject, TypedPythonObject}
-import zio.internal.{ExecutionMetrics, Executor}
+import zio.internal.Executor
 import zio.blocking.{Blocking, effectBlocking}
+import zio.duration._
 import zio.{Has, RIO, Runtime, Task, UIO, ZIO}
 import zio.interop.catz._
 
 import scala.collection.JavaConverters._
-import scala.collection.mutable.ListBuffer
 import scala.concurrent.ExecutionContext
+import scala.reflect.io.Directory
```

Comment:
Can you remove this import if it's no longer used?

Reply:
Done!